common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -10,13 +11,13 @@
 </div>
 <div class="annotation-body">
 
-    <div class="block block-highlight">${text}</div>
-    <div class="block block-comment">${comment}</div>
+    <div class="block block-highlight">${text | n, unicode}</div>
```

Comment:
`unicode` now needs to be `decode.utf8`.  I assume these all already contain HTML?